db/models.py

```diff
@@ -99,6 +99,14 @@ class RoleModifier(Base):
     def __repr__(self):
         return f"RoleModifier(id={self.id!r}, guild_id={self.guild_id!r}, role_id={self.role_id!r}, modifier={self.modifier!r})"
 
+class MorningPoints(Base):
+    __tablename__ = "morning_points"
+    user_id = Column(BigInteger, primary_key=True)
+    timestamp = Column(DateTime, default=func.now(), onupdate=func.now())
+    weekly_count = Column(Integer, nullable=False, default=0)
```

Comment:
Can we rename this to `daily_count` since it accrues once per day rather than once per week?

Reply:
it resets once a week

Reply:
But it counts the number of days since last reset, right?

Reply:
My main issue here is if I saw `weekly_count=2` I would expect that to mean that the user has said good morning for two consecutive weeks